Safari not working on SauceLabs

# Bug
Safari not working on saucelabs with configuration:

```
desired-capabilities.run-type=sauce
desired-capabilities.run-platform=mac
desired-capabilities.browser-type=safari
```

This results in sauce launching MacOS 10.10 with Safari 8, displaying

`Caused by: org.openqa.selenium.WebDriverException: Infrastructure Error -- The Sauce VMs failed to start the browser or device.`
in the console.

The sauce platform configurator https://saucelabs.com/platform/platform-configurator#/ specifies sending "macos 12" instead which launches Safari 15.

Setting the `desired-capabilities.browser-version=15` causes the following error:

`Caused by: org.openqa.selenium.WebDriverException: Misconfigured -- Unsupported OS/browser/version/device combo: OS: 'Mac 10.10', Browser: 'safari', Version: '15.', Device: 'unspecified', Screen Resolution: '1600x1200' (WARNING: The server did not provide any stacktrace information)`

# Expected
using the config

```
desired-capabilities.run-type=sauce
desired-capabilities.run-platform=mac
desired-capabilities.browser-type=safari
```

should launch a safari browser on the newest version of mac, or alternatively offer mac_12 or monterey under Platform

# Repro

set the following desired capabilities:

```
desired-capabilities.run-type=sauce
desired-capabilities.run-platform=mac
desired-capabilities.browser-type=safari
```

then run a test that makes use of the sauce webdriver

--- a/models/src/main/java/io/github/kgress/scaffold/models/enums/desktop/Platform.java
+++ b/models/src/main/java/io/github/kgress/scaffold/models/enums/desktop/Platform.java
@@ -18,6 +18,7 @@ public enum Platform {
   XP("XP"),
   Vista("VISTA"),
   Mac("MAC"),
+  Monterey("macOS 12"),
   Linux("LINUX"),
   Unix("UNIX");
 

